### PR TITLE
update RBAC assertions

### PR DESCRIPTION
**Pull request by praveen4g0**
* `RBAC` assertions update.
* Exposed event listeners routes cleanup upon test completion.

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ Tag\|TagB                    |have either TagA or TagB.
```
Comment (ppitonak) [inline README.md:158]: Let's change CHANNEL on line 156 to `stable`. The same applies to file `env/test/test.properties`

**Review by ppitonak — CHANGES_REQUESTED** [review]
Something is still wrong with olm.spec. I tried installation
`CATALOG_SOURCE=redhat-operators CHANNEL=stable gauge run --env "default, test" --tags "install" --log-level=debug --verbose   specs/olm.spec` and it never finished...

```
2021/05/06 14:16:45 output: subscription.operators.coreos.com/openshift-pipelines-operator-rh created
      * Subscribe to operator    ...[PASS]
      * Wait for TektonConfig CR availability    ...[PASS]
2021/05/06 14:17:27 Waiting for availability of pipelines cr [pipeline]
      * Validate pipelines deployment    ...[PASS]
2021/05/06 14:17:48 Waiting for availability of triggers cr [trigger]
2021/05/06 14:17:53 Waiting for availability of triggers cr [trigger]
2021/05/06 14:17:58 Waiting for availability of triggers cr [trigger]
      * Validate triggers deployment     ...[PASS]
2021/05/06 14:18:18 Waiting for availability of addon cr
      * Verify TektonAddons Install status       ...[PASS]
```

**Review by ppitonak — APPROVED** [review]
Actually this code runs fine so I'll merge it and update the channel in new PR.

**Pull request merged**